geomesa-core/src/main/scala/org/locationtech/geomesa/core/util/GeoMesaBatchWriterConfig.scala

```diff
@@ -0,0 +1,57 @@
+package org.locationtech.geomesa.core.util
+
+import java.util.concurrent.TimeUnit
+
+import com.typesafe.scalalogging.slf4j.Logging
+import org.apache.accumulo.core.client.BatchWriterConfig
+
+import scala.util.Try
+
+object GeoMesaBatchWriterConfig extends Logging {
+  val WRITERLATENCYSECONDS = "geomesa.batchwriter.latency.seconds"  // Measured in seconds
+  val WRITERLATENCYMILLIS  = "geomesa.batchwriter.latency.millis"   // Measured in millis
+  val WRITERMEMORY         = "geomesa.batchwriter.memory"           // Measured in bytes
+  val WRITERTHREADS        = "geomesa.batchwriter.maxthreads"
+  val WRITETIMEOUT         = "geomesa.batchwriter.timeout.seconds"  // Timeout measured in seconds.  Likely unnecessary.
+
+  val DEFAULTLATENCY   = 10000l   // 10 seconds
+  val DEFAULTMAXMEMORY = 1000000l // 1 megabyte
+  val DEFAULTTHREADS   = 10
+
+  protected [util] def fetchProperty(prop: String): Option[Long] =
+    for {
+      p <- Option(System.getProperty(prop))
+      num <- Try(java.lang.Long.parseLong(p)).toOption
+    } yield num
+
+  protected [util] def buildBWC: BatchWriterConfig = {
+    val bwc = new BatchWriterConfig
+
+    fetchProperty(WRITERLATENCYSECONDS) match {
+      case Some(latency) =>
+        logger.trace(s"GeoMesaBatchWriter config: maxLatency set to $latency seconds.")
+        bwc.setMaxLatency(latency, TimeUnit.SECONDS)
+      case None =>
+        val milliLatency = fetchProperty(WRITERLATENCYMILLIS).getOrElse(DEFAULTLATENCY)
+        logger.trace(s"GeoMesaBatchWriter config: maxLatency set to $milliLatency milliseconds.")
+        bwc.setMaxLatency(milliLatency, TimeUnit.MILLISECONDS)
+    }
+
+    val memory = fetchProperty(WRITERMEMORY).getOrElse(DEFAULTMAXMEMORY)
+    logger.trace(s"GeoMesaBatchWriter config: maxMemory set to $memory bytes.")
+    bwc.setMaxMemory(memory)
```

Comment:
@jnh5y does this work with the strings ```100M``` or ```300k```  ? If not can we open a ticket to support that in config similar to Java opts? I think there is some accumulo code or hadoop code somewhere we can nab.